model/stripe/model.sql

```diff
@@ -0,0 +1,132 @@
+create or replace view {{env.schema}}.stripe_invoices as (
+
+  select
+    customer,
+    date,
```

Comment:
@jthandy This is may be a minor thing, but I don't think we should be using date as a column name because it is also a data type in Redshift. invoice_date works